Bottom: 66776790bc00581076897210accfa8e1edfc3402
Top:    74c09dc125f9c5411cf9625097f5613f89a02bfe
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-28 23:48:01 -0400

Set the MPC5200 i2s driver to BROKEN status.

Set the MPC5200 i2s driver to BROKEN status.

Signed-off-by: Jon Smirl <jonsmirl@gmail.com>


---

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index 9fc9082..e7dd79a 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -21,7 +21,7 @@ config SND_SOC_MPC8610_HPCD
 
 config SND_SOC_MPC5200_I2S
 	tristate "Freescale MPC5200 PSC in I2S mode driver"
-	depends on PPC_MPC52xx && PPC_BESTCOMM
+	depends on PPC_MPC52xx && PPC_BESTCOMM && BROKEN
 	select SND_SOC_OF_SIMPLE
 	select PPC_BESTCOMM_GEN_BD
 	help
